runtime-modules/common/src/working_group.rs

```diff
@@ -19,8 +19,8 @@ pub enum WorkingGroup {
     Membership,
 }
 
-/// Working group interface to use in the in the pallets with working groups.
-pub trait WorkingGroupIntegration<T: crate::Trait> {
+/// Working group interface to work with its members - workers and leaders.
+pub trait WorkingGroupParticipation<T: crate::Trait> {
```

Comment:
This name, and in fact the last, is not ideal, can we find a proper noun here? What does this actually represent?

Reply:
I renamed it to `WorkingGroupAuthenticator`.